packages/x-components/src/components/base-rating.vue

```diff
@@ -85,14 +85,14 @@
     display: inline-block;
```

Comment:
I think we should also remove this `inline-block`, and replace the `span` that we render by a `div`, which defaults to `block`. This way we should also avoid problems related to the line-height.